### ISSUE TITLE
Support compressed tifs for read/write

### ISSUE THREAD

**Issue by freeman-lab** [thunder-project/thunder#239]
In the refactor currently underway (https://github.com/thunder-project/thunder/pull/237) we'll add a `totif` method that saves an `Images` object to tif files. @d-v-b can you post here the code snippet that adds optional compression when using `sklearn.external.tifffile`?

**Comment by d-v-b** [comment]
compression is passed as an integer kwarg to the `imsave()` function, e.g. `imsave(im_path, image, compress=1)`. The default, `compress = 0` means no compression, and increasing values of `compress` above 1 increase the level of compression, which I think is done via zlib.

More generally, there are other kwargs that people might want to use when saving tifs, since there's plenty of metadata that can be set in the tif format. How possible is it to completely wrap the functionality of `tiffffile` in thunder?

**Comment by freeman-lab** [comment]
Ok great thanks @d-v-b ! Generally it'd be easy to add whatever kwargs we want, because the `thunder.images` `totif` method is more or less just calling the `tifffile` `imsave` method on every record. One option is to list them and pass them along explicitly, the other is to just to pass all extra `kwargs` from `totif`  to `imave` . I sort of prefer the former because it can be more explicitly documented.

**Comment by d-v-b** [comment]
Agreed @freeman-lab. What do you think about more complex cases of thunder wrapping some function from another module that has kwargs? `timeSeries.normalize(baseline=window-fast)` comes to mind, where `scipy.ndimage.filters.percentile_filter` is called with `mode = nearest`, but the user might want something else... (`mode = nearest` causes strong edge effects). How should this kind of case be handled?
